components/worktemplates/components/customize.tsx

```diff
@@ -0,0 +1,119 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import styled from 'styled-components';
+import {useIntl} from 'react-intl';
+
+import PublicPrivateSelector from 'components/widgets/public-private-selector/public-private-selector';
+import {Visibility} from '@mattermost/types/worktemplates';
+import Input from 'components/widgets/inputs/input/input';
+
+export interface CustomizeProps {
+    className?: string;
+    name: string;
+    visibility: Visibility;
+
+    onNameChanged: (name: string) => void;
+    onVisibilityChanged: (visibility: Visibility) => void;
+}
+
+const Customize = ({
+    name,
+    visibility,
+    onNameChanged,
+    onVisibilityChanged,
+    ...props
+}: CustomizeProps) => {
+    const {formatMessage} = useIntl();
+
+    return (
+        <div className={props.className}>
+            <div className='name-section-container'>
+                <p>
+                    <strong>
```

Comment:
❤️ love the use of semantic html, but not sure if this is the place for strong or should be just <b>